code/lib/preview-api/src/modules/store/csf/normalizeArrays.ts

```diff
@@ -0,0 +1,4 @@
+export const normalizeArrays = <T>(array: T[] | T | undefined): T[] => {
+  if (Array.isArray(array)) return array;
+  return typeof array === 'function' ? [array] : [];
```

Comment:
Is this only intended to be used for functions and arrays of functions?  Maybe the generic should be locked down just a bit, then?  Since it might be unexpected to get back `[]` from calling this with `normalizeArrays('foo')` (I'd expect to get back `['foo']`)

Reply:
I think the function check is not neccesary actually, and we didn't do that before, so I just made the function more generic.